kotest/src/main/kotlin/com/baeldung/kotest/beforetest/Database.kt

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.kotest.beforetest
+
+
+data class User(val id: Long, val username: String)
+
+class Database {
```

Comment:
Stylelistic, but would it make more sense to call this the `UserService` or `UserRepository` class?